src/server/auth/server/api_server.go

```diff
@@ -820,58 +509,56 @@ func (a *apiServer) AuthorizeInTransaction(
 	txnCtx *txnenv.TransactionContext,
 	req *auth.AuthorizeRequest,
 ) (resp *auth.AuthorizeResponse, retErr error) {
-	if !a.isActive() {
-		return nil, auth.ErrNotActivated
+	if err := a.isActive(); err != nil {
+		return nil, err
 	}
 
-	callerInfo, err := a.getAuthenticatedUser(txnCtx.ClientContext)
-	if err != nil {
+	if err := a.expiredEnterpriseCheck(txnCtx.ClientContext); err != nil {
```

Comment:
Per the comment above about keeping pipelines running, I think this `expiredEnterpriseCheck` would be important to remove (I think the expired enterprise check should be removed from the other non-Authenticate() APIs as well, but I think the reason we never noticed they were there in tests is that pipelines don't call those APIs).

I'm surprised that `TestPipelinesRunAfterExpiration` passed—we may need to debug that test too